front-end/public/eventPage.js

```diff
@@ -71,10 +61,10 @@ chrome.contextMenus.onClicked.addListener((clickData) => {
   }
 
   chrome.notifications.create({
-    title: "Click Notifier",
+    title: "Text Saved",
     message:
-      "You clicked [" + matchingIds[0].name + "]. The text is: '" + clickData.selectionText + "'",
-    iconUrl: "Text Savvy Logo.png",
+      "The text has been added to your workspace: " + clickData.selectionText + ".'",
```

Comment:
I thought of not adding the test on the notification because, in case of having a long text saved, it would be a long notification.

What I can do is to trim it down, wrap it around quotation mark and att ellipses at the end, like I did when rendering the text in the textbox.